src/radical/pilot/utils/prof_utils.py

```diff
@@ -89,20 +89,16 @@
             'exec_rp'     : [{ru.EVENT: 'exec_start'             },
                              {ru.EVENT: 'cu_start'               }],
             'exec_sh'     : [{ru.EVENT: 'cu_start'               },
-                             {ru.EVENT: 'cu_exec_start'          }],
-            'exec_prte'   : [{ru.EVENT: 'cu_exec_start'          },
                              {ru.EVENT: 'prte_init_complete'     }],
             'prte_phase_1': [{ru.EVENT: 'prte_init_complete'     },
                              {ru.EVENT: 'prte_sending_launch_msg'}],
             'prte_phase_2': [{ru.EVENT: 'prte_sending_launch_msg'},
-                             {ru.EVENT: 'prte_running'           }],
-            'exec_app'    : [{ru.EVENT: 'prte_running'           },
-                             {ru.EVENT: 'prte_iof_complete'      }],
-            'prte_phase_3': [{ru.EVENT: 'prte_iof_complete'      },
+                             {ru.EVENT: 'app_start'              }],
+            'exec_app'    : [{ru.EVENT: 'app_start'              },
+                             {ru.EVENT: 'app_stop'               }],
+            'prte_phase_3': [{ru.EVENT: 'app_stop'               },
```

Comment:
Seeing that in our group we tend to call the user-facing code 'application' and the executable of a CU 'kernel', should we name this `krnl` or `kern` instead of `app`?

Reply:
How about `task`?

Reply:
We went for cmd

Reply:
True - but at the moment our traces have `app_*` events.  But I don't mind changing that right now - the experiments are done, and we are the only ones using the `app/cmd` traces right now.